### PR TITLE
Make files executable in mac

### PR DESCRIPTION
**Pull request by isaac73**
# Description and Motivation 

Make files executable in mac

# Change type 

- [x] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

**Checklist:**

- [x] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: scripts/git/fix-perms.sh [hunk 1]
```diff
@@ -13,32 +13,38 @@ cd $APP_HOME
 chmod ugo+x ./scripts/build
 chmod ugo+x ./scripts/git/setup
 chmod ugo+x ./scripts/git/*.sh
+chmod ugo+x ./scripts/containers.sh
+chmod ugo+x ./scripts/env-vars-setup
+chmod ugo+x ./scripts/env-vars-setup.cmd
```
Comment (dluc) [inline scripts/git/fix-perms.sh]: remove

File: scripts/git/fix-perms.sh [hunk 2]
```diff
@@ -13,32 +13,38 @@ cd $APP_HOME
...
+chmod ugo+x ./Config/run.sh
 chmod ugo+x ./ASAManager/run.sh
 chmod ugo+x ./Telemetry/run.sh
 
 git update-index --chmod=+x ./scripts/build
 git update-index --chmod=+x ./scripts/git/setup
 git update-index --chmod=+x ./scripts/git/*.sh
+git update-index --chmod=+x ./scripts/containers.sh
+git update-index --chmod=+x ./scripts/env-vars-setup
+git update-index --chmod=+x ./scripts/env-vars-setup.cmd
```
Comment (dluc) [inline scripts/git/fix-perms.sh]: remove

File: scripts/git/fix-perms.sh (hunk 1 above)
Comment (isaac73) [reply scripts/git/fix-perms.sh]: done

File: scripts/git/fix-perms.sh (hunk 2 above)
Comment (isaac73) [reply scripts/git/fix-perms.sh]: done

**Pull request merged**